charts/hocs-outbound-proxy/templates/hocs-proxy-networkpolicy.yaml

```diff
@@ -8,15 +8,12 @@ spec:
     matchLabels:
       role: hocs-outbound-proxy
   policyTypes:
-    - Ingress
-    - Egress
+  - Ingress
+  - Egress
```

Comment:
Is `Egress` necessary if you're specifying empty?

Reply:
Yes because there is a more restrictive policy being applied in default-deny-all